jsx/FilterForm.js

```diff
@@ -132,7 +132,7 @@ class FilterForm extends React.Component {
       } else {         // null and undefined handled here
         filter[key].value = '';
       }
-      filter[key].exactMatch = (type === "SelectElement");
+      filter[key].exactMatch = (type === "SelectElement" || type === "select");
```

Comment:
Is there a situation where `type === "SelectElement"` ?

Reply:
maybe? i couldn't find any personally but i also don't want to break anything :p